packages/amazon-cognito-identity-js/src/CognitoUser.js

```diff
@@ -1206,9 +1206,36 @@ export default class CognitoUser {
 		return undefined;
 	}
 
+	/**
+	 * PRIVATE ONLY: This is an internal only method and should not
+	 * be directly called by the consumers.
+	 */
+	createGetUserRequest() {
+		return this.client.promisableRequest('GetUser', {
```

Comment:
```suggestion
		return this.client.promisifyRequest('GetUser', {
```
I think this needs to be same name as method in client. Typo?